### PR TITLE
MSDKUI-1722: [Android] Component app findings

### PR DESCRIPTION
**Pull request by imvjk**
GuidanceStreetLabelView -- Moved "Getting your location" out of view.
GuidanceStreetLabelView -- Remove outer spacing
GuidanceStreetLabelView - added option for without street name in dev app
GuidanceStreetLabelView -- component is centered now

Signed-off-by: Vijay Yadav <5671997+imvjk@users.noreply.github.com>

### CODE REVIEW

File: MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_street_label_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<com.here.msdkui.guidance.GuidanceStreetLabelView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/guidanceStreetLabelViewId"
+    android:layout_width="wrap_content"
+    android:layout_height="40dp"
```
Comment (tstaszak89) [inline MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_street_label_fragment.xml]: If layout height is not in style, user will not be able to change height of this component. User should not be able to set height of this component?

File: MSDKUIDev/app/src/main/java/com/here/msdkuidev/component/GuidanceStreetLabel.kt [hunk 2]
```diff
@@ -32,5 +34,8 @@ class GuidanceStreetLabel : AppCompatActivity() {
         title = setting.title.toLowerCase()
         setContentView(resourceId)
         if (!setting.defaultView) guidanceStreetLabelView.setCurrentStreetData(setting.guidanceStreetLabelData)
+        if(setting.redText) {
+            guidanceStreetLabelView.findViewById<TextView>(R.id.guidance_current_street_text).setTextColor(Color.RED)
```
Comment (tstaszak89) [inline MSDKUIDev/app/src/main/java/com/here/msdkuidev/component/GuidanceStreetLabel.kt]: Why not customized by style?

File: MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_street_label_fragment.xml (hunk 1 above)
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_street_label_fragment.xml]: this is app part but anyways i wil fix it :)

File: MSDKUIDev/app/src/main/java/com/here/msdkuidev/component/GuidanceStreetLabel.kt (hunk 2 above)
Comment (imvjk) [reply MSDKUIDev/app/src/main/java/com/here/msdkuidev/component/GuidanceStreetLabel.kt]: i think its good to have customization some by style and some like this to showcase all possibilities.

**Pull request merged**